challenges/functions.js

```diff
@@ -7,25 +7,38 @@
   * In the body of the function return the callback with the two parameters that you created
 */
 
+function consume(param1, param2, cb) {
+    return cb(param1, param2);
+}
+
 
 /* Step 2: Create several functions to callback with consume();
   * Create a function named add that returns the sum of two numbers
   * Create a function named multiply that returns the product of two numbers 
   * Create a function named greeting that accepts a first and last name and returns "Hello first-name last-name, nice to meet you!"
 */
+function add(x, y) {
+    return (x + y);
+}
+
+function multiply(x, y) {
+    return (x * y);
+}
 
+function greeting(firstName, lastName) {
+    return (`Hello ${firstName} ${lastName}, nice to meet you!`);
+}
 
 /* Step 3: Check your work by uncommenting the following calls to consume(): */
-//consume(2,2,add); // 4
-// consume(10,16,multiply); // 160
-// consume("Mary","Poppins", greeting); // Hello Mary Poppins, nice to meet you!
-
+consume(2, 2, add); // 4
```

Comment:
Using the console log will print out the results for each of these function calls.

`console.log(consume(2,2,add));`